automated-comments/go/general/avoid_printing_and_logging.md

```diff
@@ -0,0 +1,8 @@
+Usually it is not a good idea to output something to `stdout` or `stderr` in lower level packages.
+That is done in higher level or the `main` package, that have the necessary context to for example know what to do with an error.
+Using `log.Println` or `fmt.Println` in a lower level package can not be overwritten from the outside.
+Best avoid them in the exercises here. They are useful for debugging but should be removed before submitting.
```

Comment:
```suggestion
Printing and logging are useful for debugging, but should be removed before submitting an exercise, or releasing a package.
```

Or something like that. The overal advice here is _great_, and you probably want to make sure it's clear that you basically never want it, unless you're writing an application (if I understood correctly).

Reply:
I like this suggestion.